server/app/app.py

```diff
@@ -85,15 +85,12 @@ def static_redirect(dataset, therest):
     return redirect(f"/static/{therest}", code=301)
 
 
-def favicon_redirect(dataset):
-    """ redirect favicon to static dir """
-    return redirect("/static/favicon.png", code=301)
-
-
 def dataroot_index():
     # FIXME with a splash screen that includes a listing of all the datasets.
-    # or perhaps a login screen if this is a hosted environment
-    data = "<H1>Welcome to cellxgene</H1>"
+    # or perhaps a login screen if this is a hosted environment,
+    # or have a configuration option to redirect to a user specified page.
+    data = '<!doctype html><html lang="en"><head>'
```

Comment:
this doesn't look right - there is no closing tag for <head>.

If it is empty, you can remove it, or use <head/>

Reply:
Thanks, I just pushed a fix for <head>